src/components/tools/text-toolbar.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { useFloatingToolbarLocation } from "./hooks/use-floating-toolbar-location";
+import { TextToolbarButton } from "./text-toolbar-button";
+import { IRegisterToolApiProps } from "./tool-tile";
+import { isMac } from "../../utilities/browser";
+
+import "./text-toolbar.sass";
```

Comment:
`text-style-bar.*` => `text-toolbar.*`

The rendering is essentially unchanged aside from converting to a functional component and using the `useFloatingToolbarLocation()` hook. The rendering code will likely change substantially when the new design is implemented.